Algorithm.CSharp/OpeningBreakoutAlgorithm.cs

```diff
@@ -13,6 +13,10 @@
  * limitations under the License.
 */
 
+// plotting and logging controls
+#define ENABLE_PLOTTING
+#undef ENABLE_ORDER_UPDATE_LOGGING
```

Comment:
Although it's exactly the same, I personally find it more readable to have a commented #define instead:
// #define ENABLE_ORDER_UPDATE_LOGGING